cerberus/tests/test_validation.py

```diff
@@ -1,14 +1,18 @@
 # -*- coding: utf-8 -*-
 
-import re
-import sys
+
+from bson import decimal128
```

Comment:
Sorry, I did not get this point: since it is a functionality offered by Cerberus, it should be tested within it. But I let you take the decision over it